samples/CommunityToolkit.Maui.Sample/ViewModels/Extensions/KeyboardExtensionsViewModel.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CommunityToolkit.Maui.Core.Platform;
+using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace CommunityToolkit.Maui.Sample.ViewModels.Extensions;
+public partial class KeyboardExtensionsViewModel : BaseViewModel
+{
+
+	public KeyboardExtensionsViewModel()
+	{
+		ShowKeyboard = new Command<ITextInput>(OnShowKeyboard);
+		HideKeyboard = new Command<ITextInput>(OnHideKeyboard);
+		IsKeyboardShowing = new Command<ITextInput>(OnIsKeyboardShowing);
+	}
+
+	void OnIsKeyboardShowing(ITextInput view)
+	{
+		if (!view.IsSoftKeyboardShowing())
+		{
+			KeyboardShowingText = $"Soft Input Is Currently Hidden";
+		}
+		else
+		{
+			KeyboardShowingText = $"Soft Input Is Currently Showing";
+		}
+	}
+
+	async void OnHideKeyboard(ITextInput view)
```

Comment:
Use RelayCommand attribute and convert to async Task

Reply:
ohhh fancy!!